### ISSUE TITLE
About UNIVERSE++ training cost

### ISSUE THREAD

**Issue by SaoYear** [line/open-universe#7]
Hi, thanks for the great work and codes.

I have some questions about the training efforts of the UNIVERSE++ model. I am now using a customized static dataset (whose size is approximate to the universe speech enhancement datasets in your paper) to train the UNIVERSE++ model. 

Here are my questions:
1. the default training steps in the paper is 1,500,000 steps for universe speech enhancement, but I found that in this repo, the maximum training steps is set to [600,000](https://github.com/line/open-universe/blob/939dd3c12acaac9da5c45d241437169cca3e647a/config/trainer/default.yaml#L5) for VBD dataset. Considering my customized dataset large size, does this default `max_steps` guarantees a converged performance? 
2. I am now using 2 nvidia-A100 to train the model. The training speed is about 5,000 steps/GPU hour, which means the entire training (600,000 steps) cost 120 A100 GPU hours to converge, is this a normal number ?

Many thanks in advance!

**Comment by fakufaku** [comment]
Hi @SaoYear , what you are describing sound right. The 1500000 was for the larger dataset in the paper.
Because VBD is a lot smaller, I could decrease the number of steps.
120 gpu-hours sounds about right for 600000 steps.

**Comment by SaoYear** [comment]
Many thanks for your quick response : ) 
To train the model with 1.5 M steps, I think the lr scheduler setup should be changed accordingly. Could you please provide the model config file if possible? As [this file](https://github.com/line/open-universe/blob/939dd3c12acaac9da5c45d241437169cca3e647a/config/model/default.yaml) seems to be valid for 600k-step training only.

**Comment by SaoYear** [comment]
According to the USE training setup, I modifed the `default.yaml ` with the factors below, leaving the rest unchanged:
```
score_model:
  rate_factors: [2, 3, 5, 8]
  n_channels: 48
scheduler:
  generator:
    T_warmup: 50000
    T_cosine: 1000000
  discriminator:
    T_warmup: 50000
    T_cosine: 1000000
```



> Many thanks for your quick response : ) To train the model with 1.5 M steps, I think the lr scheduler setup should be changed accordingly. Could you please provide the model config file if possible? As [this file](https://github.com/line/open-universe/blob/939dd3c12acaac9da5c45d241437169cca3e647a/config/model/default.yaml) seems to be valid for 600k-step training only.

**Comment by fakufaku** [comment]
Unfortunately, I don't have access to the original config files anymore.
The change you are suggesting looks good. Just make sure that `T_max` is set properly. It should be set to the number of steps by default.
https://github.com/line/open-universe/blob/939dd3c12acaac9da5c45d241437169cca3e647a/config/model/universepp_24k.yaml#L107C7-L107C34

**Comment by SaoYear** [comment]
Thank you for the response!
